frontend/src/Order.tsx

```diff
@@ -12,12 +12,15 @@ import { AccessorySelector } from './components/AccessorySelector'
 import { OrderBehalfofPicker } from './components/OrderBehalfofPicker'
 import { useValidPositionsAsync } from './utils/hooks'
 import { apiBackend } from './api/apiClient'
+import { SubmitFormDialog } from './components/SubmitFormDialog'
 import { FieldHeader } from './components/FieldHeader'
 
 const Order = () => {
     const api = new apiBackend()
 
+    const [isSubmitEnabled, setIsSubmitEnabled] = useState(false)
     const [resultRitm, setResultRitm] = useState('')
+    const [submitPopupOpen, setSubmitPopupOpen] = useState(false)
```

Comment:
Maybe it should be `const [isSubmitPopupOpen, setIsSubmitPopupOpen] = useState(false)` such that it matches the naming of the state on line 21?